gazebo_ros_control:
  pid_gains:
    LF_WHEEL_JOINT:
      p: 100.0
    RF_WHEEL_JOINT:
      p: 100.0
    LH_WHEEL_JOINT:
      p: 100.0
    RH_WHEEL_JOINT:
      p: 100.0

  # Base frame_id
  base_frame_id: base_link

  # Odometry fused with IMU is published by robot_localization, so
  # no need to publish a TF based on encoders alone.
  enable_odom_tf: false

  # Smb hardware provides wheel velocities
  estimate_velocity_from_position: false

  # Wheel separation and radius multipliers
  wheel_separation_multiplier: 1.0 # default: 1.0
  wheel_radius_multiplier    : 1.0 # default: 1.0

  # Velocity and acceleration limits
  # Whenever a min_* is unspecified, default to -max_*
  linear:
    x:
      has_velocity_limits    : true
      max_velocity           : 1.0   # m/s
      has_acceleration_limits: true
      max_acceleration       : 0.2   # m/s^2
  angular:
    z:
      has_velocity_limits    : true
      max_velocity           : 1.0   # rad/s
      has_acceleration_limits: true
      max_acceleration       : 0.2   # rad/s^2

smb_diff_drive:
  left_wheel: ['LF_WHEEL_JOINT', 'LH_WHEEL_JOINT']
  right_wheel: ['RF_WHEEL_JOINT', 'RH_WHEEL_JOINT']